pkgs/applications/misc/vhs/default.nix

```diff
@@ -14,12 +14,12 @@ buildGoModule rec {
   vendorHash = "sha256-9nkRr5Jh1nbI+XXbPj9KB0ZbLybv5JUVovpB311fO38=";
 
   nativeBuildInputs = [ installShellFiles makeWrapper ];
-  buildInputs = [ ttyd ffmpeg ];
+  buildInputs = [ ttyd ffmpeg chromium ];
```

Comment:
As the programs are only used explicitly in the wrapper below, no need to add them to buildInputs.

Reply:
@NickCao Thanks, I removed them, tested and pushed again.